fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7326,8 +7326,11 @@ class KeyValueStoreRedwood : public IKeyValueStore {
 		                   : FLOW_KNOBS->SIM_PAGE_CACHE_4K)
 		        : FLOW_KNOBS->PAGE_CACHE_4K;
 		// Rough size of pages to keep in remap cleanup queue before being cleanup.
-		int64_t remapCleanupWindowBytes = (BUGGIFY ? deterministicRandom()->randomInt64(0, 100) * 1024 * 1024
-		                                           : SERVER_KNOBS->REDWOOD_REMAP_CLEANUP_WINDOW_BYTES);
+		int64_t remapCleanupWindowBytes = SERVER_KNOBS->REDWOOD_REMAP_CLEANUP_WINDOW_BYTES;
+		if (g_network->isSimulated() && BUGGIFY)
+			remapCleanupWindowBytes = (deterministicRandom()->randomInt(0, 2) == 0)
```

Comment:
`deterministicRandom()->randomInt(0, 2) == 0` is equivalent to `deterministicRandom()->coinflip()` but also the usual way we express "true with some probability p" is `deterministicRandom()->random01() < p`